urls.py

```diff
@@ -0,0 +1,41 @@
+from django.conf import settings
+from django.conf.urls import include, patterns, url
+from django.views.i18n import javascript_catalog
+from django.contrib import admin
+
+import workbench.urls
+
+admin.autodiscover()
+
+# Packages to include in the JavaScript i18n strings
+JS_INFO_DICT = {
+    'packages': ('ubcpi.xblock',),
+}
+
+urlpatterns = patterns(
+    '',
+    # Django built-in
+    url(r'^admin/', include(admin.site.urls)),
+
+    # Provided by XBlock
+    url(r'^/?', include(workbench.urls)),
+
+    # edx-ora2 apps
+    # url(r'^peer/evaluations/', include(openassessment.assessment.urls)),
+
+    # JavaScript i18n
+    (r'^jsi18n/$', 'django.views.i18n.javascript_catalog', JS_INFO_DICT),
```

Comment:
as far as I can tell you aren't using any of hte i18n stuff but you've copied it all in here throughout. Are you planning to internationalize this block at some point?